boards/arm/frdm_k64f/dts.fixup

```diff
@@ -29,3 +29,12 @@
 #define CONFIG_WNCM14A2A_GPIO_MDM_SEND_OK_NAME		NXP_KINETIS_UART_4006C000_WNCM14A2A_MDM_SEND_OK_GPIOS_CONTROLLER
 #define CONFIG_WNCM14A2A_GPIO_MDM_SEND_OK_PIN		NXP_KINETIS_UART_4006C000_WNCM14A2A_MDM_SEND_OK_GPIOS_PIN
 #endif
+
+#if ETH_PORT == 0
+#define CONFIG_ETH_MCUX_0_NAME		ETH_LABEL
+#define CONFIG_ETH_MCUX_0_IRQ_PRI		ETH_IRQ_PRIORITY
+
+#define CONFIG_ETH_MCUX_0_MAC3		NXP_MCUX_ETHERNET_400C0004_LOCAL_MAC_ADDRESS_3
+#define CONFIG_ETH_MCUX_0_MAC4		NXP_MCUX_ETHERNET_400C0004_LOCAL_MAC_ADDRESS_4
+#define CONFIG_ETH_MCUX_0_MAC5		NXP_MCUX_ETHERNET_400C0004_LOCAL_MAC_ADDRESS_5
```

Comment:
It would be better to move the ethernet fixups to `soc/arm/nxp_kinetis/k6x/dts.fixup`